source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -369,6 +369,30 @@ progress.
 For more information on DDL operations, see
 :ref:`txn-prod-considerations-ddl`. 
 
+Network Latency
+~~~~~~~~~~~~~~~
+
+Network latency or long physical distances between migration components
+can negatively affect sync speed.
+
+Latency between ``mongosync`` and destination shards
+  For each operation on the source cluster, ``mongosync`` does two 
+  roundtrips to the destination server. The larger the latency, the 
+  slower the sync.
+
+Latency between destination shards
+  Mongosync writes operations and updates its own metadata on the 
+  destination cluster in batches in a transaction. This can result in 
+  cross-shard transactions, which may be more costly if the shards are 
+  far apart.
+
+Latency between the nodes of each individual replica set on the source or destination
```

Comment:
```suggestion
Latency between the nodes of any replica set (including shard backing replica set) on the source or destination cluster
```

Reply:
Updated, but added the parenthetical info to the body of the text to keep the title length manageable.